### DISCUSSION TITLE
Creation of an user form to update service log

### DISCUSSION THREAD

**Discussion by SMetriq** [Budibase/budibase#4456] [Help]
Hi,

I followed steps on budibase.com and I have now two table (vehicles and service log). I would like to create an user form to change values on a selected service log or vehicle.

I created an user form and I updated parameter on "Update" but not sure to understand how to feed the user form with pre existing values, how can I do this ? 

Thanks

**Comment by mjashanks** [comment]
Form inputs all have a "Default Value" setting - is that what you are looking for?

This will prefill the input upon loading the form.
